pkg/logs/client/destination.go

```diff
@@ -74,3 +85,35 @@ func (d *Destination) Send(payload []byte) error {
 
 	return nil
 }
+
+// SendAsync sends a message to the destination without blocking. If the queue is full, the incoming messages will be
+// dropped
+func (d *Destination) SendAsync(payload []byte) {
+	d.once.Do(func() {
+		inputChan := make(chan []byte, chanSize)
+		d.inputChan = inputChan
+		go d.consumeAsync()
+	})
+
+	select {
+	case d.inputChan <- payload:
+	default:
+		// TODO: Display the warning in the status
+		if d.warningCounter%warningPeriod == 0 {
```

Comment:
I think we should do `d.warningCounter > 0 && d.warningCounter%warningPeriod == 0 `

Reply:
I think you want to catch it on the first occurence though